core/plugins/lazy/lazy.py

```diff
@@ -0,0 +1,150 @@
+from talon import Module, Context, actions
+
+mod = Module()
+
+# when lazy main menu is not shown
+ctx_off = Context()
+ctx_off.matches = r"""
+app: neovim
+and not win.title: /FILETYPE:\[lazy\]/
+"""
+
+# when lazy main menu is shown
+ctx_on = Context()
+ctx_on.matches = r"""
+app: neovim
+and win.title: /FILETYPE:\[lazy\]/
+"""
+
+
+# https://github.com/folke/lazy.nvim?tab=readme-ov-file#-usage
+@mod.action_class
+class Actions:
+    def lazy_home():
+        """open lazy home menu"""
+
+    def lazy_install():
+        """install missing lazy plugins"""
+
+    def lazy_update():
+        """update lazy plugins"""
+
+    def lazy_sync():
+        """run lazy install, clean and update"""
```

Comment:
Something seems off with this line? Not sure if it would actually error

Reply:
Can you explain what seems off?

Reply:
Seems like it's just a visual rendering bug on mobile, but the docstring shows up on a new line. I'm guessing if you don't see it, it's actually fine so will accept

Reply:
Ya is probably that :)